src/test/lambda/models/samTemplates.test.ts

```diff
@@ -72,9 +71,11 @@ describe('getSamCliTemplateParameter', function () {
     })
 
     it('should return error if the template option is not valid', async function () {
-        const errorMessage = `${repromptUserForTemplate} is not valid sam template`
-        const error = await assertThrowsError(async () => getSamCliTemplateParameter(repromptUserForTemplate))
-        assert.strictEqual(error.message, errorMessage, 'Should fail for same error')
+        await assert.rejects(
```

Comment:
Should this be `assert.throws`? `getSamCliTemplateParameter` is not async.

Reply:
Yup must've missed it.